packages/api/src/graphql/mutation.private.ts

```diff
@@ -548,6 +550,22 @@ export const GraphQLAdminMutation = new GraphQLObjectType<undefined, Context>({
       }
     },
 
+    // Comment
+    // =======
+
+    createComment: {
+      type: GraphQLNonNull(GraphQLComment),
+      args: {input: {type: GraphQLNonNull(GraphQLCommentInput)}},
+      async resolve(root, {input}, {authenticate, dbAdapter}) {
+        const {roles} = authenticate()
+        authorise(CanCreateComment, roles)
+
+        return dbAdapter.comment.createComment({
```

Comment:
`await`

Reply:
@bishoymelek why did you mark this as resolved? This still is missing an `await`

Reply:
fixed